sos/report/plugins/__init__.py

```diff
@@ -479,6 +495,11 @@ def timeout(self):
             return _timeout
         return self.plugin_timeout
 
+    def set_timeout_hit(self):
+        self._timeout_hit = True
+        self.manifest.add_field('end_time', time())
```

Comment:
Doesnt it matter we store in `end_time` a different format (`time.time()`) than for non-timeouted plugins (`datetime.datetime.now()`)?

Is it worth adding also `run_time`?

Reply:
Oh, good catch! Yup, this should all use `datetime`. Adding a `run_time` field sounds good too